Constants/index.js

```diff
@@ -0,0 +1,13 @@
+const constants  = {
```

Comment:
give meaningful name and use SCREAMING_SNAKE_CASE for naming variables in constants